data api/src/main/java/de/tinf15b4/kino/api/KinoWebDataService.java

```diff
@@ -103,12 +104,13 @@ private static void jettyEnableInherit(Server server) {
     @Bean
     public CommandLineRunner loadData(DataInitializer initializer) {
         return (args) -> {
-            initializer.initialize();
+            // initializer.initialize();
         };
     }
 
     public static void main(String[] args) {
-        // HACK: Passing java system properties is hard, environment variables are easy
+        // HACK: Passing java system properties is hard, environment variables
+        // are easy
```

Comment:
I dont know why... I though my formatting settings were correct. Let me check that